tests/test_github_start_date.py

```diff
@@ -33,7 +34,8 @@ def test_run(self):
 
         # run the test for all the streams excluding 'events' stream
         # as for 'events' stream we have to use dynamic dates
-        self.run_test('2020-04-01T00:00:00Z', '2021-10-08T00:00:00Z', self.expected_streams() - {'events'})
+        self.run_test('2020-04-01T00:00:00Z', '2022-01-27T00:00:00Z', self.expected_streams() - {'events', 'pull_requests', 'pr_commits', 'review_comments', 'reviews'})
```

Comment:
Can we define the start dates in variables such as start_date1 or something?

Reply:
Assigned date in variable date_1 & date_2